src/site/content/en/blog/css-props-and-vals/index.md

```diff
@@ -0,0 +1,206 @@
+defined.
+
+Second, unlike standard properties, registered properties aren't validated when
+they're parsed. Rather they're validated when they're computed. That means both
+that invalid values won't appear as invalid when inspecting the element's
+properties, and including an invalid property after a valid one won't fall back
+to the valid one; an invalid property will, however, fall back to the registered
+property's default.
+
+## Animating custom properties
+
+A registered custom property provides a fun bonus beyond type checking: the
+ability to animate it! A basic animation example looks like this:
+
+```html
+<script>
+CSS.registerProperty({
+  name: '--stop-color',
+  syntax: '<color>',
+  inherits: false,
+  initialValue: 'blue',
+});
+</script>
+
+<style>
+button {
+  --stop-color: red;
+  transition: --stop-color: 1s;
+}
+
+button:hover {
+  --stop-color: green;
+}
+</style>
+```
+
+When you hover over the button, it'll animate from red to green! Without
+registering the property, it'll jump from one color to the other Because,
+without being registered, fhe browser doesnt know what to expect between one
+value and the next and therefore can't guarentee the ability to transition themß.
+This example can be taken a step further, though, to animate CSS gradients! The
+following CSS can be written with the same registered property:
+
+```css
+button {
+  --stop-color: red;
+  background: linear-gradient(var(--stop-color), black);
+  transition: --stop-color: 1s;
+}
+
+button:hover {
+  --stop-color: green;
+}
+```
+
+This will animate our custom property that's part of the `linear-gradient`, thus
+animating our linear gradient. Check out the Glitch below to see the full code
+in action and play around with it yourself.
+
+<!-- Copy and Paste Me -->
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    allow="geolocation; microphone; camera; midi; vr; encrypted-media"
+    src="https://glitch.com/embed/#!/embed/houdini-props-and-vals?path=style.css&previewSize=40&attributionHidden=true"
+    alt="houdini-props-and-vals on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+## Conclusion
+
+Houdini [is on its way](http://ishoudinireadyyet.com/) to browsers, and with it,
+entirely new ways of working with and extending CSS. With the [Paint
+API](https://developers.google.com/web/updates/2018/01/paintapi) already shipped
+and now Custom Props and Vals, our creative toolbox is expanding, allowing us to
+define typed CSS properties and use them to create and animate new and exciting
+designs. There's more on the way, too, in the [Houdini issue
+queue](https://github.com/w3c/css-houdini-drafts/issues) where you can give
+feedback and see what's next for Houdini. Houdini exists to develop features
+that explain the "magic" of styling and layout on the web, so get out there and
+put those magical features to good use.
+
+Photo by Maik Jonietz on Unsplash
```

Comment:
```suggestion
_Photo by Maik Jonietz on Unsplash._
```

Can you link to the image and link to unsplash? See the example at the bottom of this post: https://web.dev/samesite-cookies-explained

Reply:
This was throwing: `warning  Don’t use literal URLs without angle brackets  no-literal-urls  remark-lint`. There was nothing in it that wasn't in the samesite cookies source. I used the `--no-verify` flag to submit.

Reply:
I think it throws that warning if you paste the actual URL on the page without putting it inside of a markdown link.

Reply:
Took a look at the PR and I agree that seems weird. You can add a line above that snippet to tell the linter to chill out:

```
<!--lint disable no-literal-urls-->
```

I can look into why it's doing that later.